packages/migrations/package.json

```diff
@@ -11,7 +11,7 @@
         "build": "tsc -b",
         "build:ci": "yarn build",
         "clean": "shx rm -rf lib ${npm_package_config_snapshot_name} ${npm_package_config_snapshot_name}-*.zip",
-        "lint": "tslint --format stylish --project .",
+        "lint": "tslint --format stylish --project . && prettier --check 'src/**/*.{ts,tsx,json,md}' --config ../../.prettierrc",
```

Comment:
It seems like you agreed to keep `prettier` as a separate script in the `asset-swapper` package, can we do the same here?

Reply:
Addressed in https://github.com/0xProject/0x-monorepo/pull/2541/commits/1da8f6887177629b9c6099dad84f8c348113a4ab